src/config/global-config.c

```diff
@@ -419,6 +420,22 @@ int Read_Global(const OS_XML *xml, XML_NODE node, void *configp, void *mailp)
             if (Config) {
                 Config->memorysize = atoi(node[i]->content);
             }
+        /* update check system */
+        else if (strcmp(node[i]->element, xml_update_check) == 0) {
```

Comment:
The new block was accidentally introduced inside the `xml_memorysize` block.
So I'm always getting this error message

```
ERROR: (1230): Invalid element in the configuration: 'update_check'.
```

Reply:
Fixed !